lib/bundler/settings.rb

```diff
@@ -32,6 +32,7 @@ class Settings
       silence_root_warning
       unlock_source_unlocks_spec
       update_requires_all_flag
+      path_relative_to_cwd
```

Comment:
(you'll also need to document the setting in `bundle-config.ronn`, as this will cause `spec/quality_spec` to fail)